openreview/conference/templates/programchairWebfield.js

```diff
@@ -1460,7 +1460,9 @@ var displayPaperStatusTable = function() {
       conferenceStatusData.filteredRows = filteredRows
     }
     renderTable(container, filteredRows);
-    $(container + ' .btn-export-data').text(`Export ${filteredRows.length} records`)
+    $(container + " .btn-export-data").text(
+      "Export ".concat(filteredRows.length, " records")
```

Comment:
Generally if you aren't using template strings you would use `+` for string concatenation